src/Radicle/Internal/Eval.hs

```diff
@@ -222,11 +232,14 @@ callFn f arguments = case f of
     evalManyWithEnv :: Env Value -> NonEmpty Value -> Lang m Value
     evalManyWithEnv env exprs =
         NonEmpty.last <$> withEnv (const env) (traverse baseEval exprs)
-    argumentBindings :: [Ident] -> Lang m (Env Value)
-    argumentBindings names =
+    posArgBindings :: [Ident] -> Lang m (Env Value)
```

Comment:
To reduce the number of cases above, I would keep this as one function `argumentBindings :: LambdaArgs -> Lang m (Env Value)`.

Reply:
Great suggestion. Updated.